public/js/index.js

```diff
@@ -77,6 +77,23 @@ function istorn2020Counter() {
     const timerMinutes = Math.floor(diffSeconds % 3600 / 60).toFixed(0).padStart(2, "0");
     const timerHours = Math.floor(diffSeconds % 86400 / 3600).toFixed(0).padStart(2, "0");
     const timerDays = Math.floor(diffSeconds / 86400).toFixed(0);
+    
+    if (timerSeconds == 60){
+      timerSeconds = 0;
```

Comment:
You're gonna need to change the `const` above to `var` or `let`, otherwise this will error

Reply:
ah yeah, didn't spot it was contansts